pkg/northbound/gnmi/set.go

```diff
@@ -55,13 +56,13 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 	//Update
 	for _, u := range req.GetUpdate() {
 		target := u.Path.GetTarget()
-		targetUpdates[target] = s.doUpdateOrReplace(u, targetUpdates)
+		targetUpdates[target], err = s.doUpdateOrReplace(u, targetUpdates)
```

Comment:
the error here assigned bt not checked until line 87, effectively letting `onos-config` try to apply a partial state/config. I think we should check for errors at line 73. and if any return. 
```	
if err != nil {
		return nil, status.Error(codes.Internal, err.Error())
}
```

Reply:
done